packages/components/psammead-story-promo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-story-promo",
-  "version": "2.5.3",
+  "version": "2.5.4",
```

Comment:
This isnt a patch as it adds functionality

Reply:
Changed it to a minor change